vendor/opam/src/core/gen/gen.ml

```diff
@@ -0,0 +1,98 @@
+(* trivial cppo replacement with just enough features to do conditional compilation *)
```

Comment:
You'll need to modify bootstrap to make this work. Moreover, for this to be usable in bootstrap it cannot use use `Re`.

Reply:
I've removed the whole thing upstream.